### DISCUSSION TITLE
混合笔记法

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#36] [备发布站的评论系统]
# 混合笔记法

Rethinking my PKM part 4: How do you organize your notes? - YouTube[[🧑zsolt]]分析了[[🧑Nick Milo]]和[[🧑Tiago Forte]]对笔记组织的不同人群不同偏好的类型划分后，总结出了自己的类型划分想法，且通过视觉的方式呈现，...

https://notes.oldwinter.top/%E6%B7%B7%E5%90%88%E7%AC%94%E8%AE%B0%E6%B3%95

**Comment by Neal-799** [comment]
在b站查，有这个视频观看，学习到很多谢谢博主
